packages/lexical-table/src/LexicalTableUtils.ts

```diff
@@ -856,6 +856,9 @@ export function $computeTableMapSkipCellCheck(
       $isTableRowNode(row),
       'Expected TableNode children to be TableRowNode',
     );
+    if (row.getChildrenSize() === 0) {
+      tableMap[rowIdx] = [];
+    }
```

Comment:
I don't think this is correct regarding rowspan, but you can initialize `startMapRow` here which will have the same effect and should be correct (at least with regard to that constraint). You'll have to remove the initialization below but I can't put a suggestion on code you didn't change in the review

```suggestion
    const startMapRow = getMapRow(rowIdx);
```